src/lib/crypto/kimchi_bindings/stubs/src/caml/caml_bytes_string.rs

```diff
@@ -0,0 +1,26 @@
+use ocaml::{FromValue, IntoValue, Runtime, Value};
+use ocaml_gen::{const_random, Env, OCamlDesc};
+
+pub struct CamlBytesString<'a>(pub &'a [u8]);
```

Comment:
I think I can live with this :) can you add some doc to explain why we have this struct here? Otherwise perfect!